src/utils/user.js

```diff
@@ -0,0 +1,3 @@
+export const getUserId = () => {
+  return localStorage.getItem("userId");
+}
```

Comment:
I'm not sure if we need to do any null checks here, since we would pass a null into the methods in service (e.g. `getCollection`)